### PR TITLE
better docs for ActionView::ViewPaths#prepend_view_path

### PR DESCRIPTION
**Pull request by alexanderkustov**
First time using docsdoctor, improving some documentation of the methods that got deprecated but no docs were updated.

### CODE REVIEW

File: actionview/lib/action_view/view_paths.rb [hunk 1]
```diff
@@ -46,10 +46,22 @@ def details_for_lookup
...
+    #
+    # ==== Parameters
+    # * <tt>path</tt> - If a String is provided, it gets converted into
+    #   the default view path. You may also provide a custom view path
+    #   (see ActionView::PathSet for more information)
     def append_view_path(path)
       lookup_context.view_paths.push(*path)
     end
-
+
```
Comment (vipulnsward) [inline actionview/lib/action_view/view_paths.rb]: ✂️ extra line

**Comment by maclover7** [comment]
@alexanderkustov Can you please squash your commits? Thanks!

**Comment by alexanderkustov** [comment]
@maclover7 commits squashed

**Comment by vipulnsward** [comment]
Thanks! 
@alexanderkustov in future, please add a `[ci skip]` in commit message for doc-only changes, so that we don't run tests for them on Travis.

**Pull request merged**